charts/README.md

```diff
@@ -92,10 +92,10 @@ helm delete `<RELEASE NAME>` -n `<RELEASE NAMESPACE>`
 
 | Repository | Name | Version |
 |------------|------|---------|
-|  | openebs-crds | 4.0.0 |
```

Comment:
why does crds chart version need to change?

Reply:
We're mapping the chart version to the crds chart.

Reply:
and why do we do that?

Reply:
Hmm, not sure. Without a change to the crds chart, is it right to increment its version?

Reply:
If we add a future template/functionality which makes use of the chart's version then that template would have to work with version ranges which differ from the chart's version. Which is not too hard to do I suppose. Do you think I should revert to 4.0.0 for the crds chart?

Reply:
IIRC we also made the crd chart follow the mayastor chart version.
But I agree that we may not need to always bump the crd if there are no changes..

Reply:
Reverted openebs-crds to 4.0.0.